configurator.py

```diff
@@ -10,12 +10,13 @@
 The code in this script then overrides the globals()
 
 I know people are not going to love this, I just really dislike configuration
-complexity and having to prepend config. to every single variable. If someone
+complexity and having to prepend config to every single variable. If someone
 comes up with a better simple Python solution I am all ears.
 """
 
 import sys
 from ast import literal_eval
+from types import NoneType
```

Comment:
It will work only in Python 3.10+ ( https://docs.python.org/3.10/whatsnew/3.10.html#types ) which sounds like unnecessary limitation